src/apps/bristleback_apps/aanderaa/user_code/user_code.cpp

```diff
@@ -162,36 +189,52 @@ void setup(void) {
   configASSERT(userConfigurationPartition);
   userConfigurationPartition->getConfig("currentAggPeriodMin", strlen("currentAggPeriodMin"),
                                         current_agg_period_min);
+  userConfigurationPartition->getConfig("nSkipReadings", strlen("nSkipReadings"),
+                                        n_skip_readings);
+  userConfigurationPartition->getConfig("readingIntervalMs", strlen("readingIntervalMs"),
+                                        reading_interval_ms);
+  userConfigurationPartition->getConfig("payloadWdToS", strlen("payloadWdToS"),
+                                        payload_wd_to_s);
+  userConfigurationPartition->getConfig("plUartBaudRate", strlen("plUartBaudRate"),
+                                        baud_rate);
```

Comment:
Just to confirm — since these are user configs, they won't be sent to the Sofar backend through Spotter. Is that the behavior we want? If we want to have remote visibility into these values, they need to be system configs.

Reply:
Good catch - I'll move these to sys partition and retest.